### PR TITLE
Disable pull up resistors of all GPIOS

### PR DESCRIPTION
**Pull request by NIwkoe**
According to Toradex forum, bits 5-6 controls the PULL settings of GPIO on the imx8:
00b - prohibited
01b - pull-up
10b - pull-down
11b - pull disabled

Since all GPIOs on the RCU have external pull resistors, we decided to turn off all internal ones. Set bits 5-6 to 0b11 for all GPIOs.

### CODE REVIEW

**Review by HatsyRei — APPROVED** [review]

**Review by karhin1990 — APPROVED** [review]
Again, I would urge for code documentation. 
It's hard to recall back why some pin config are `0x06000021` while some are `0x06000061` after few months later.

Your PR description is very clear and helpful, I would put it inside the dtsi as a comment!

**Comment by kengsooncheah** [comment]
Is turning off all pull-ups intentional?

File: recipes-kernel/linux/files/imx8-apalis-smartracks.dtsi [hunk 1]
```diff
@@ -344,87 +344,95 @@
...
-                IMX8QM_M41_GPIO0_00_LSIO_GPIO0_IO12         0x06000021 /* MXM3 5   */
-                IMX8QM_UART0_RTS_B_LSIO_GPIO0_IO22          0x06000021 /* MXM3 6   */
-                IMX8QM_MLB_CLK_LSIO_GPIO3_IO27              0x06000021 /* MXM3 63  */
-                IMX8QM_M41_GPIO0_01_LSIO_GPIO0_IO13         0x06000021 /* MXM3 7   */
-                IMX8QM_UART0_CTS_B_LSIO_GPIO0_IO23          0x06000021 /* MXM3 8   */
+                /* Bits 5:6 controls the PULL settings of GPIO on the iMX8:        */
+                /* 0b00 - prohibited                                               */
+                /* 0b01 - pull-up                                                  */
+                /* 0b10 - pull-down                                                */
+                /* 0b11 - pull disabled                                            */
```
Comment (karhin1990) [inline recipes-kernel/linux/files/imx8-apalis-smartracks.dtsi:355]: Do put a comment on why are we disabling all GPIOs internal pull as well.

Comment (NIwkoe) [reply recipes-kernel/linux/files/imx8-apalis-smartracks.dtsi:355]: Still working with HW team to verify, I found out actually in Rev A all GPIOs internal pulls are turned off, so we thought of doing the same.

Comment (NIwkoe) [reply recipes-kernel/linux/files/imx8-apalis-smartracks.dtsi:355]: HW team had verified the GPIOs. We will turn off all pull resistors, I've added comment to describe it.

**Comment by NIwkoe** [comment]
Since I made changes after getting approval, I will only merge after getting thumbs up from you guys. I've changed the PR title and amended the comment. @kengsooncheah @karhin1990

**Pull request merged**